utilities/rpp-unittests/OCL/BatchPD_ocl.cpp

```diff
@@ -0,0 +1,729 @@
+    {
+        percentage[i] = 100;
+    }
+
+    cl_mem d_input, d_input_second, d_output;
+    cl_platform_id platform_id;
+    cl_device_id device_id;
+    cl_context theContext;
+    cl_command_queue theQueue;
+    cl_int err;
+    err = clGetPlatformIDs(1, &platform_id, NULL);
+    err = clGetDeviceIDs(platform_id, CL_DEVICE_TYPE_GPU, 1, &device_id, NULL);
+    theContext = clCreateContext(0, 1, &device_id, NULL, NULL, &err);
+    theQueue = clCreateCommandQueue(theContext, device_id, 0, &err);
+    d_input = clCreateBuffer(theContext, CL_MEM_READ_ONLY, ioBufferSize * sizeof(Rpp8u), NULL, NULL);
+    d_input_second = clCreateBuffer(theContext, CL_MEM_READ_ONLY, ioBufferSize * sizeof(Rpp8u), NULL, NULL);
+    d_output = clCreateBuffer(theContext, CL_MEM_READ_ONLY, ioBufferSize * sizeof(Rpp8u), NULL, NULL);
+    err = clEnqueueWriteBuffer(theQueue, d_input, CL_TRUE, 0, ioBufferSize * sizeof(Rpp8u), input, 0, NULL, NULL);
+    err = clEnqueueWriteBuffer(theQueue, d_input_second, CL_TRUE, 0, ioBufferSize * sizeof(Rpp8u), input_second, 0, NULL, NULL);
+    rppHandle_t handle;
+
+    rppCreateWithStreamAndBatchSize(&handle, theQueue, noOfImages);
+
+    clock_t start, end;
+    double cpu_time_used;
+    start = clock();
+    string test_case_name;
+    switch (test_case)
+    {
+    case 0:
+        test_case_name = "contrast";
+        rppi_contrast_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, newMin, newMax, noOfImages, handle);
+        break;
+    case 1:
+        test_case_name = "jitter";
+        rppi_jitter_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, kernelSize, noOfImages, handle);
+        break;
+    case 2:
+        test_case_name = "blur";
+        rppi_blur_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, kernelSize, noOfImages, handle);
+        break;
+    case 3:
+        test_case_name = "brightness";
+        rppi_brightness_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, alpha, beta, noOfImages, handle);
+        break;
+    case 4:
+        test_case_name = "blend";
+        rppi_blend_u8_pkd3_batchPD_gpu(d_input, d_input_second, srcSize, maxSize, d_output, alpha, noOfImages, handle);
+        break;
+    case 5:
+        test_case_name = "color_temperature";
+        rppi_color_temperature_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, adjustmentValue, noOfImages, handle);
+        break;
+    case 6:
+        test_case_name = "gamma_correction";
+        rppi_gamma_correction_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, gamma, noOfImages, handle);
+        break;
+    case 7:
+        test_case_name = "fog";
+        rppi_fog_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, fogValue, noOfImages, handle);
+        break;
+    case 8:
+        test_case_name = "snow";
+        rppi_snow_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, snowPercentage, noOfImages, handle);
+        break;
+    case 9:
+        test_case_name = "lens_correction";
```

Comment:
Codacy found an issue: [Variable 'test_case_name' is assigned a value that is never used.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/rpp/pullRequest?prid=5976402)